main/main.cpp

```diff
@@ -204,7 +204,8 @@ void finalize_physics() {
 
 void Main::print_help(const char *p_binary) {
 
-	print_line(String(VERSION_NAME) + " v" + get_full_version_string() + " - https://godotengine.org");
+	print_line(String(VERSION_NAME) + " v" + get_full_version_string() + " - " + String(VERSION_WEBSITE));
+	OS::get_singleton()->print("Free and open source software under the terms of the MIT license.\n");
```

Comment:
Do we want to add this as a constant too? LICENSE_HINT or something? Because if we make "Godot" changeable, *this* should be easily changeable/discoverable too IMO :)

Reply:
I don't think it's worth it.

To be fair, I also debated whether it was worth making the URL configurable (or include it at all). In the end I chose to do it, with this simple use case in mind: people who make slight modifications of the engine and want to label their custom build as such. For example if someone made a build with e.g. Admob, Facebook and Google login and Firebase built in, they could easily identify it as "Godot Moniez Edition" and link to godotmoniez.com.

For any rebranding that would go beyond that and as far as changing the license (taking into account that the MIT license for the existing code cannot be revoked, but one can distribute their Godot flavour under a proprietary license as long as they follow the terms of the MIT for the upstream code), a lot more changes would be needed than what is currently possible, and this should be discussed in its own proposal and see specific developments made for it (if there's any actual interest in it).